modules/code-editor/src/backend/typings.d.ts

```diff
@@ -9,6 +7,7 @@ export interface TypingDefinitions {
 export type FileType =
   | 'action_legacy'
   | 'action_http'
+  | 'action' // New signature without arguments
```

Comment:
That's because there are different "signatures" for each of them. http actions doesn't have the sdk, while action doesn't have the wrapper to use arguments. The file type is mostly for that